src/posix/client.cpp

```diff
@@ -68,11 +68,14 @@ namespace {
 struct Config
 {
     const char *mNetifName;
+    uint64_t    mTimeout;
 };
 
 enum
 {
     kLineBufferSize = OPENTHREAD_CONFIG_CLI_MAX_LINE_LENGTH,
+    kUsPerSec       = 1000000,
+    kNsPerUs        = 1000,
```

Comment:
How about using `static constexpr`?  

The use of `enum` to define constants was primarily a solution employed before the OpenThread stack transitioned to C++11.